src/zui/ZUIOrganizeSidebar/index.tsx

```diff
@@ -241,14 +262,60 @@ const ZUIOrganizeSidebar = (): JSX.Element => {
                 zIndex: 1000,
               }}
             >
+              {recentOrganizations.filter((org) => org?.id != orgId).length >
+                0 &&
```

Comment:
I would create a variable for this with a descriptive name, because I think it's quite difficult to read what it means here. Something like `hasRecentOrganizations` or something.

Reply:
Makes absolute sense, it's not easy to read this way!